### PR TITLE
Update types

### PR DESCRIPTION
**Pull request by woomurf**
- update types
- fix some return value

### CODE REVIEW

**Review by liayoo — APPROVED** [review]
LGTM, hthanks!

**Comment by woomurf** [comment]
Thanks quick review!

File: src/types.ts [hunk 1]
```diff
@@ -21,22 +36,44 @@ export interface Account {
...
+
 export interface RewardInstance {
   id?: string;
   rewardTypeId: string;
-  type: RewardType;
+  category: RewardTypeCategory;
   amount: number;
   distributeAt: RewardDistributeType;
-  distributeCronTime?: any;
+  // distributeCronTime?: any;
```
Comment (platfowner) [inline src/types.ts:72]: remove or comments?

File: src/types.ts [hunk 2]
```diff
@@ -9,7 +9,22 @@ export enum RewardType {
 export enum RewardDistributeType {
   ON_ACTIVITY = 'ON_ACTIVITY',
   MANUAL = 'MANUAL',
-  CRON = 'CRON',
+  // CRON = 'CRON',
```
Comment (platfowner) [inline src/types.ts:12]: remove or comments?

**Review by platfowner — COMMENTED** [review]
LGTM with some comments.
Thanks!

File: src/types.ts (hunk 2 above)
Comment (woomurf) [reply src/types.ts:12]: Fix in #12 ! Thanks to @liayoo !!

**Pull request merged**